contrib/dev.Dockerfile

```diff
@@ -19,6 +19,7 @@ RUN apt-get update
 RUN apt-get install apt-utils -y
 RUN apt-get update
 RUN apt-get install gettext python3-pip -y
+RUN apt-get install postgresql-client -y
```

Comment:
A bit of nitpicking, but strictly speaking the client is not required since you can just `docker exec -it db bash` to use the postgresql client that's inside the database container.

However this does make development much friendlier. Maybe add this in a separate commit instead so it's not implied that postgresql-client is **required** to run `manage.py loaddata`?